### PR TITLE
CI: Upgrade Emscripten to 2.0.25

### PR DESCRIPTION
**Pull request by akien-mga**
That's the version used by current containers for Godot 3.4.

Not a candidate for cherry-picking to `3.3` as we will stick to version 2.0.15 to avoid surprises.

### CODE REVIEW

**Comment by object71** [comment]
```diff
diff --git a/servers/audio_server.h b/servers/audio_server.h
--- servers/audio_server.h
+++ servers/audio_server.h
@@ -244,9 +244,9 @@
 		AudioCallback callback;
 		void *userdata;
 
 		bool operator<(const CallbackItem &p_item) const {
-			return (callback == p_item.callback ? userdata < p_item.userdata : callback < p_item.callback);
+			return (callback == p_item.callback ? userdata < p_item.userdata : (void*)callback < (void*)p_item.callback);
 		}
 	};
 
 	Set<CallbackItem> callbacks;

```

**Comment by object71** [comment]
@akien-mga This should fix the build.

**Comment by akien-mga** [comment]
Only for a single occurrence, but there are dozens more. I already added a flag to disable those warnings, but haven't rebased this PR yet.

File: scene/gui/rich_text_label.cpp [hunk 1]
```diff
@@ -449,9 +447,7 @@ int RichTextLabel::_process_line(ItemFrame *p_frame, const Vector2 &p_ofs, int &
 					fh = line_ascent + line_descent;
 
 					if (end && c[end - 1] == ' ') {
-						if (p_mode == PROCESS_CACHE) {
-							spaces_size += font->get_char_size(' ').width;
-						} else if (align == ALIGN_FILL) {
```
Comment (akien-mga) [inline scene/gui/rich_text_label.cpp:454]: Might be a bug that this was set but unused (as in, maybe it's meant to be used). It's no longer in the `master` branch, but that code was refactored.

**Pull request merged**